service-pdf/docker/app/Dockerfile

```diff
@@ -27,7 +27,7 @@ RUN if [[ $ENABLE_XDEBUG = 1 ]] ; then \
 COPY --chown=root:root service-pdf/composer.json /app/composer.json
 COPY --chown=root:root service-pdf/composer.lock /app/composer.lock
 RUN curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/bin --filename=composer --version 2.4.1 \
-    && composer install --prefer-dist --no-interaction --no-cache --no-scripts --optimize-autoloader -d /app \
+    && composer install --prefer-dist --no-interaction --no-cache --no-scripts --no-dev --optimize-autoloader -d /app \
     && chown -R root:root /app/vendor \
     && rm /app/composer.json \
     && rm /usr/bin/composer
```

Comment:
Is there a tests directory and phpunit.xml to remove here as well?

Reply:
I have added to the dockerignore, lines to ensure it doesn't get copied over